portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase

```diff
@@ -65,4 +65,54 @@ definition {
 		}
 	}
 
+	@description = "StructuredContent can be filtered with a set priority"
+	@priority = "5"
+	test StructuredContentContainsUpdatedPriorityValueFilteredByEqualTo {
+		property portal.acceptance = "true";
+
+		task ("Given three web contents are created with default priority") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var responseToParse1 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 1");
+			var responseToParse2 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 2");
+			var responseToParse3 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 3");
+		}
+
+		task ("When edit a web content with different priorities") {
+			HeadlessWebcontentAPI.editStructuredContentWithPriority(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				priority = "1.3",
+				responseToParse = "${responseToParse2}",
+				title = "WC WebContent Title 2");
+		}
+
+		task ("When web contents are filtered with 1.3 priority") {
+			var response = HeadlessWebcontentAPI.filterStructuredContentWithPriority(filtervalue = "priority%20eq%201.3");
+		}
+
+		task ("Then the response body includes the 1.3 priority field") {
+			HeadlessWebcontentAPI.assertPriorityFieldIsFilterWithDefaultValue(
```

Comment:
Maybe here we can add something to make sure that the StructuredContent that is filtered is the same that we have edited? Maybe we can take advantage of the id that we already know and check whether it is the same as the edited structure

Reply:
we may also want to check whether it is the only element that appears as filtered